Assignment-1-Calculator.c

```diff
@@ -0,0 +1,130 @@
+#include<stdio.h>
+
+
+/* To check the error
+error - 1 for  invalid expression 
+error -2 for division by zero error */
+int error = 0;
+
+
+// The function is to check the priority of the operator according to the DMAS
+int priority_op(char operator){
+    if(operator == '*' || operator == '/')return 1;
+    if(operator == '+' || operator == '-')return 0;
+    return -1;
+}
+
+
+/* The fuction to  calculate the input */

```

Comment:
apply code formatting and match function names with this standards, take care of this at all places.

Use camelcase for function names.
Start function names with a verb to indicate action.
Keep names concise but meaningful.
ex :
int calculateSum(int a, int b);
void displayMessage(const char *message);